tests/check/check_signed_video_auth.c

```diff
@@ -1120,29 +1120,36 @@ START_TEST(detect_change_of_public_key)
   // |settings|; See signed_video_helpers.h.
 
   // Generate 2 GOPs
-  test_stream_t *list = create_signed_nalus("IPPIPP", settings[_i]);
-  test_stream_check_types(list, "IPPSIPP");
-
+  test_stream_t *list = create_signed_nalus("IPPIPPI", settings[_i]);
+  test_stream_check_types(list, "IPPSIPPSI");
+  test_stream_item_t *i_item = test_stream_item_remove(list, 9);
```

Comment:
I think it is more clear if you use `test_stream_pop_last_item(...)` instead. Then one does not have to count number of characters to understand that you've removed the last item.